tests/03_02_01-last_fork_active_powpeg_redeem_script.js

```diff
@@ -0,0 +1,60 @@
+const chai = require('chai');
+chai.use(require('chai-as-promised'));
+const expect = chai.expect;
+const redeemScriptParser = require('@rsksmart/powpeg-redeemscript-parser');
+const {getRskTransactionHelpers} = require('../lib/rsk-tx-helper-provider');
+const {getBridge, getLatestActiveForkName} = require('../lib/precompiled-abi-forks-util');
+const CustomError = require('../lib/CustomError');
+const {disableWhitelisting} = require('../lib/2wp-utils');
+const removePrefix0x = require('../lib/utils').removePrefix0x;
+const {getBtcClient} = require('../lib/btc-client-provider');
+const rskUtils = require('../lib/rsk-utils');
+const {
+  GENESIS_FEDERATION_ADDRESS,
+  GENESIS_FEDERATION_REDEEM_SCRIPT,
+} = require('../lib/constants');
+
+const fulfillRequirementsToRunAsSingleTestFile = async (rskTxHelper, btcTxHelper) => {
+  const latestForkName = rskUtils.getLatestForkName();
+  await rskUtils.activateFork(latestForkName);
+  await disableWhitelisting(rskTxHelper, btcTxHelper);
```

Comment:
I don't think it is necessary to disable the whitelisting in this test. There's no pegin being made.
You could simplify this function to be:

```js
const fulfillRequirementsToRunAsSingleTestFile = async () => {
  const latestForkName = rskUtils.getLatestForkName();
  await rskUtils.activateFork(latestForkName);
};
```
And remove the `const {disableWhitelisting} = require('../lib/2wp-utils');` line above.

And call it like:

```js
if (process.env.RUNNING_SINGLE_TEST_FILE) {
      await fulfillRequirementsToRunAsSingleTestFile();
    }
```

And the ` const btcTxHelper = getBtcClient();` line would not be necessary. 
And the `const {getBtcClient} = require('../lib/btc-client-provider');` line could be removed as well.